doc/source/sections/getting-started/introduction.rst

```diff
@@ -13,6 +13,22 @@ Docker MiGrid Structure
 The Docker MiGrid is build up of several different components and services.
 These have been seperated into two different categories, Associated services and MiGrid services.
 
+Example Deployments
+-------------------
+
+This repository contains three example deployments which can be used to test migrid or be extended to production use.
+
+- development
+  This is the default deployment which contains a standard migrid setup and a test account to login.
+
+- development_gdp
+  This deployment contains a configuration with all GDPR related functionalities enabled. That means the system will be
+  more closed down and depend on higher security standards.
+
+- advanced
```

Comment:
shouldn't it be `production` here as well?

Reply:
thx, fixed